### DISCUSSION TITLE
凌霞之后会有验证码短信相关业务吗

### DISCUSSION THREAD

**Discussion by QingFeng-awa** [halo-dev/halo#6983] [General]
Halo和1Panel现在都有短信相关的功能(Halo的手机号验证,1Panel的短信告警)了,但凌霞似乎还没有相关业务开展(1Panel的专业版赠送的短信额度用完我都不知道在哪购买),故提出这个讨论。
如果凌霞未来有相关业务,也期望Halo支持一键配置相关功能。

**Comment by ruibaby** [comment]
> 1Panel的专业版赠送的短信额度用完我都不知道在哪购买

正在开发
